assets/js/projects.js

```diff
@@ -45,6 +62,23 @@ document.addEventListener('DOMContentLoaded', function() {
               date: '2024-09-01',
               partner: 'The International Council on Clean Transportation',
             },
+            {
+              category: ['support and training'],
+              status: 'past',
+              imageUrl: 'assets/img/projects/ich.jpeg',
+              title: 'Advancing Hydropower Integration in Africa with PyPSA',
+              subtitle: 'Equipping Energy Professionals to Lead Africa’s Renewable Energy Transition',
+              description: 
+              'Open Energy Transition (OET) recently collaborated with the International Centre for Hydropower (ICH) \
+              to deliver a transformative workshop on Modeling the Integration of Hydropower into Modern Energy Systems \
+              for Africa. Held in Nairobi, Kenya, from August 19–23, 2024, the program brought together 23 professionals \
+              from Eswatini, Ghana, Kenya, Nigeria, South Africa, Tanzania, and Zambia. \
+              Participants gained hands-on experience with Python-based energy modeling tools, PyPSA, and PyPSA-Earth, \
```

Comment:
Not sure about the suggested change, just remember that we have used `delegates` rather than `participants`. What do you think?

```suggestion
              Delegates gained hands-on experience with Python-based energy modeling tools, PyPSA, and PyPSA-Earth, \
```

Reply:
This is great, I think you can go ahead with it.

Reply:
Thanks a lot @PriyeshGosai!